### DISCUSSION TITLE
Share your graphs

### DISCUSSION THREAD

**Discussion by Hoogkamer** [Hoogkamer/Termboard#14] [Show and tell]
When you have created a nice graph (ontology, mind map, etc) you can share it here.

- File > Export > JSON
- rename the file to ...json.txt (json extention you cannot share here)
- then share it in a new comment and add a description
- other users can import it using File > Import > JSON

**Comment by Hoogkamer** [comment]
A mind map genereated with ChatGPT on "Personal Growth" See also the making-of here: https://youtu.be/iWuM9OaCOEo

[mindmap on personal growth (3).json.txt](https://github.com/Hoogkamer/webboardapp/files/11276760/mindmap.on.personal.growth.3.json.txt)

**Comment by Hoogkamer** [comment]
[video creation.json.txt](https://github.com/Hoogkamer/webboardapp/files/11303671/video.creation.json.txt)
An example of creating a work flow diagram using ChatGPT and combining it with a mindmap. Video here: https://youtu.be/BOMDdrFoo_k

**Comment by Hoogkamer** [comment]
[atomic habits.json.txt](https://github.com/Hoogkamer/webboardapp/files/11525280/atomic.habits.json.txt)
An example with Atomic Notes about the book Atomic Habits

**Comment by robbebob77** [comment]
Overview of locations and links for the board game Ticket to Ride Legacy: Legends of the West (2023)(https://boardgamegeek.com/boardgame/390092/ticket-ride-legacy-legends-west)

[Ticket To Ride Legacy Na Spel 1.txt.json](https://github.com/Hoogkamer/webboardapp/files/13350931/Ticket.To.Ride.Legacy.Na.Spel.1.txt.json)
